agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -16,22 +16,23 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 # Defaults
 group=default
 dir="/tmp"
-
-# always notify the user, as collection can sometimes take a while
-echo "Collecting system information"
+sysinfo_opts=( all default block libvirt kernel_config sos topology )
 
 # Process options and arguments
 
-opts=$(getopt -q -o d:g: --longoptions "dir:,group:" -n "getopt.sh" -- "$@");
+opts=$(getopt -q -o d:g: --longoptions "dir:,group:,sysinfo:,check" -n "getopt.sh" -- "$@");
 if [ $? -ne 0 ]; then
 	printf "\n"
 	printf "$script_name: you specified an invalid option\n\n"
 	printf "The following are required:\n\n"
-	printf -- "\t-d str --dir=str, str = a directory where the $script_name\n"
-	printf -- "\t                        will store and process data\n"
+	printf -- "\t-d str --dir=str,     str = a directory where the $script_name\n"
+	printf -- "\t                            will store and process data\n"
 	printf "\n"
-	printf -- "\t-g str --group=str, str = a tool group used in a benchmark\n"
-	printf -- "\t                          (the default group is 'default')\n"
+	printf -- "\t-g str --group=str,   str = a tool group used in a benchmark\n"
+	printf -- "\t                            (the default group is 'default')\n"
+	printf -- "\t       --sysinfo=str, str = comma seperated values of sysinfo to be collected\n"
+	printf -- "\t                            (available: default, all, none, sos, kernel_config, libvirt, topology, block)\n"
```

Comment:
This is a second place we repeat the list of options.  Let's make a function that will emit the list of options, something like:
```
function display_sysinfo_opts {
    printf "none, default, all"
    for item in ${sysinfo_opts[*]}; do printf ", %s" $item ; done
}
```

Where `sysinfo_opts` is defined like my previous comment above?